doc/decisions/high_level_api.md

```diff
@@ -102,11 +134,13 @@ What is not so nice:
 ## Argument ##
 
 1. Very easy to get started with, to get a key needs 3 lines of codes:
-   ```c
-   KDBHL *handle = kdbhlOpen ("/sw/elektra/kdb/#0/current");
-   printf ("number /mykey is %d\n", kdbhlGetInt (handle, "/mykey"));
-   kdbhlClose (handle);
-   ```
+   
+```c
```

Comment:
I am not sure if every markdown renderer will handle this as expected. Some might start counting again from `1.` below? Not really important, so I will merge anyway.